src/main.C

```diff
@@ -13,33 +13,13 @@
 /****************************************************************/
 
 #include "BlackBearTestApp.h"
-#include "MooseInit.h"
-#include "Moose.h"
-#include "MooseApp.h"
-#include "AppFactory.h"
-#include "TensorMechanicsActionBase.h"
-
-// Create a performance log
-PerfLog Moose::perf_log("BlackBear");
+#include "MooseMain.h"
 
 // Begin the main program.
 int
 main(int argc, char * argv[])
 {
-  // Initialize MPI, solvers and MOOSE
-  MooseInit init(argc, argv);
-
-  // Register this application's MooseApp and any it depends on
-  BlackBearTestApp::registerApps();
-
-  // register inelastic strain
-  TensorMechanicsActionBase::addCartesianComponentOutput("inelastic_strain");
```

Comment:
I just saw this got merged yesterday. 

How is `addCartesianComponentOutput()` call handled now?

Reply:
...or is it not necessary anymore? I don't actually know what the story is behind that -- this was the first time I had seen it.

Reply:
My mistake, `addCartesianComponentOutput()` should stay. The MOOSE change doesn't handle `addCartesianComponentOutput()`

Reply:
@MengnanLi91 Thanks! If you don't mind adding another PR that adds that back in, that would be great. I can merge it.

Reply:
It would just go before the `Moose::main` call

Reply:
Already in, check #382